bgpd/bgp_vty.c

```diff
@@ -362,6 +362,7 @@ static int bgp_srv6_locator_unset(struct bgp *bgp)
 
 		/* refresh per-vrf tovpn_sid_locator */
 		srv6_locator_chunk_free(bgp_vrf->tovpn_sid_locator);
```

Comment:
Please make srv6_locator_chunk_free take a double pointer and have it free then set the pointer to NULL.  Then we won't have another bug in the future when someone else adds the srv6_locator_chunk_free in a different spot and the coder again forgets to set the pointer to null

Reply:
@donaldsharp Thank you for the review. I agree with your suggestions.

Since we are going to solve this issue in a completely different way, I will close this PR and open a new one.